airflow/models.py

```diff
@@ -790,7 +790,7 @@ class TaskInstance(Base, LoggingMixin):
         Index('ti_pool', pool, state, priority_weight),
     )
 
-    def __init__(self, task, execution_date, state=None):
+    def __init__(self, task, execution_date, state=None, raw=False):
```

Comment:
I would not make it part of the signature. Just set to False on init and then use "init_run_context" to set it.

Reply:
We'd have to add `raw` as a method to `init_run_context()`, yeah?

Reply:
parameter I hope

```
def init_run_context(self, raw=False):
```

Reply:
Er, yeah that. Not a method. (I don't even know what that would look like)

Reply:
Done. Note that on v1-9-stable there isn't a `init_run_context()` so we'll have to do something else there. Perhaps just set `ti.raw = args.raw` directly in cli.py?

Reply:
Not yet there indeed, but will most likely merge those changes into 1.9-test and then stable eventually (1.9.1) or in 1.10/2.0